examples/inference_triton/README.md

```diff
@@ -1,137 +1,28 @@
 
-Merlin Inference API is designed for a seamless integration between NVTabular and Triton IS, where Triton simplifies the deployment of AI models at scale in
-production. This integration makes online data transformation and generation of the query results for new data pretty easy. Users use `NVTabular` to preprocess training data and collect statistics (in our example for categorical features) at the training phase on training data. After this transformation and fit process, NVTabular’s workflow object is serialized to be loaded at inference phase to transform the new dataset with the collected statistics. By deserializing the saved NVTabular workflow, these statistics are applied to transform the new dataset at inference phase: whenever a new query is received by the inference server, the data is first fed into NVTabular to call the transform of the NVTabular workflow object. The output is then passed to the ML/DL model (a Tensorflow model in our example) and prediction results for the query are generated accordingly.
+## Getting Started 
 
+Under this directory, we provide six example notebooks: 
 
-# Getting Started 
+- [movielens-TF](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-TF/movielens-TF.ipynb)  
+- [movielens-inference](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-TF/movielens_inference.ipynb)
+- [movielens-multihot-TF](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-TF/movielens-multihot-TF.ipynb)  
+- [movielens-multihot-inference](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-TF/movielens-multihot-inference.ipynb)
+- [movielens-HugeCTR](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-HugeCTR/movielens-HugeCTR.ipynb)
+- [movielens-HugeCTR-inference](https://github.com/NVIDIA/NVTabular/tree/main/examples/inference_triton/inference-HugeCTR/movielens-HugeCTR-inference.ipynb) 
 
-In order to use Merlin Inference API, there are two containers that the user needs to build and launch. The first one is for preprocessing with NVTabular and training a model. The other one is for serving/inference. 
+## Pulling the Merlin Docker Containers:
 
-## 1. Pulling the NVTabular Docker Container:
+In order to run the example notebooks above with Merlin Inference API, there are three containers that the user needs to build and launch:
```

Comment:
Users don't need to build the containers - maybe just leave at 'that the user needs' ? 
```suggestion
In order to run the example notebooks above with Merlin Inference API, there are three containers that the user needs:
```